src/index.js

```diff
@@ -1,42 +1,87 @@
 import "./styles.css";
+import React, { useEffect, useState } from "react";
+import ReactDOM from "react-dom";
 import { getNextGameboard, isFieldValueAlive } from "./gameboard";
 
-function renderGameboard(gameboard) {
-  return `<div class="gameboard">${gameboard
-    .map(
-      (row) =>
-        `<div class="gameboard-row">${row
-          .map(
-            (field) =>
-              `<div class="gameboard-field"><div class="gameboard-cell gameboard-cell--${
-                isFieldValueAlive(field) ? "alive" : "dead"
-              }"></div></div>`
-          )
-          .join("")}</div>`
-    )
-    .join("")}</div>`;
+// function renderGameboard(gameboard) {
```

Comment:
would advise against out commented code in the production code base, also here it seems like you are not needing it again in the future and you can always check the history with git :-)